src/common/GamePlatform.ts

```diff
@@ -74,5 +74,14 @@ export const DefaultGamePlatforms: GamePlatformMap = {
 		exeFile: 'Among Us.exe',
 		translateKey: 'platform.microsoft',
 	},
+}
+
+export const DefaultLinuxGamePlatforms: GamePlatformMap = {
+	[GamePlatform.STEAM]: {
+		available: false,
+		key: GamePlatform.STEAM,
+		run: '/usr/bin/steam steam://rungameid/945360',
+		translateKey: 'platform.steam',
+	}
```

Comment:
This should probably still have a launchType of URI. EXE isn't meant to be a windows .exe file but rather just an executable file in general so linux may also have an executable type.